apps/neuvector/neuvector/ithc/ithc.yaml

```diff
@@ -7,6 +7,16 @@ metadata:
 spec:
   values:
     neuvector:
+      controller:
+        ingress:
+          enabled: true
+          path: "/"
```

Comment:
Is this needed here? I thought only in `apps/neuvector/neuvector/neuvector.yaml` should be good.

Reply:
aat00 doesn't have a controller ingress so i put in cluster files (ithc.yaml etc). I guess moving this to neuvector.yaml and specifying "controller: ingress: enabled: false" in aat00.yaml might be a cleaner option?